flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionVertex.java

```diff
@@ -590,6 +604,15 @@ public Execution resetForNewExecution(final long timestamp, final long originati
 
 				this.currentExecution = newExecution;
 
+				synchronized (this.inputSplits){
+					for (InputSplit split: this.inputSplits){
```

Comment:
General formatting/style, this line (and others) lack some whitespaces, e.g. `split :` and `) {`. Please have another look at the formatting.

Reply:
Done.